component-classes/index.js

```diff
@@ -207,14 +207,14 @@ export const expandable = {
   expandableBleed: box.bleed,
   chevron: 'inline-block align-middle i-text-$color-expandable-icon',
   chevronNonBox: 'relative left-8',
-  chevronBox: 'absolute right-16',
+  chevronBox: '',
   chevronTransform: 'transform transition-transform transform-gpu ease-in-out',
   chevronExpand: '-rotate-180',
   chevronCollapse: 'rotate-180',
   expansion: 'overflow-hidden',
   expansionNotExpanded: 'h-0 invisible',
   button: buttonReset + ' hover:underline focus-visible:underline',
-  buttonBox: 'w-full text-left relative inline-flex items-center ' + box.box,
+  buttonBox: 'w-full text-left relative inline-flex items-center justify-between ' + box.box,
   paddingTop: 'pt-0',
   title: 'flex justify-between items-center',
```

Comment:
In React and Elements we are wrapping text and chevron in a div, which then holds `expandable.title` classes. In order for the chevron to be positioned at the end of a box Expandable there, we'll need to apply `w-full` here.

Reply:
[08e6137](https://github.com/warp-ds/css/pull/89/commits/08e613745b78b2b8ad64c6d0820316afb52f152a)